### ISSUE TITLE
0.8.3 is JupyterLab 4.x only

### ISSUE THREAD

**Issue by jhgoebbert** [jupyterlab-contrib/spellchecker#135]
You might want to set the requirement for 0.8.3 to JupyterLab>=4 as `@codemirror/language` doesn't in JupyterLab 3.x  
At lease I see this error in the WebConsole:
`Error: Shared module @codemirror/language doesn't exist in shared scope default`
https://github.com/jupyterlab-contrib/spellchecker/blob/master/src/index.ts#L5

**Comment by krassowski** [comment]
Yes, 0.8.x requires JupyterLab 4 or Notebook 7. I am not sure how to specify dependency in that case; I do not want to add `jupyterlab>=4` because notebook users do not necessarily want to install jupyterlab with this extension. Any suggestions?

**Comment by jhgoebbert** [comment]
Ah, I see 🤔 . That is tricky.

**Comment by krassowski** [comment]
Actually we can just pin jupyterlab since notebook 7 depends on jupyterlab. #138 does that.

**Comment by krassowski** [comment]
@jhgoebbert a new release (v0.8.4) with a pin is up on PyPI, can you confirm whether installation resolves correctly for you (or whether we need to yank the older versions)?

**Comment by jhgoebbert** [comment]
Hi @krassowski
yes, I installed spellchecker==0.8.4 with JupyterLab 4.0 (from source and wheel). It works perfect and I cannot see any issues at install or runtime.  
I also tried to install it with JupyterLab 3.6 using notebook 6 - now the installation failed as expected. That is much better than 0.8.3 which installs fine but fails at runtime. Yes, perhaps you really want to yank 0.8.3. That would help the users who try to install it as extension for JupyterLab 3.6.
